New translation: accessibility-statement.md (#191)
gatsbyjs/gatsby-pt-BR: docs/accessibility-statement.md

---

> title: Declaração de acessibilidade

Reviewer: ```suggestion
title: Declaração de Acessibilidade
```

---

_Última atualização: Abril 8, 2019_

> Nos queremos que todo usuário do Gatsby sinta-se bem-vindo e sinta a experiência de recompensa: isso inclui desenvolvedores e usuários com dificuldades especiais em sites, documentações e recursos do Gatsby.

Reviewer 2: ```suggestion
Nós queremos que todo usuário do Gatsby sinta-se bem-vindo e sinta a experiência de recompensa: isso inclui desenvolvedores e usuários com dificuldades especiais em sites, documentações e recursos do Gatsby.
```

> Essa página foi criada para coletar de forma simples informações sobre todo o ecossistema Gatsby em um único lugar, melhorando os canais de comunicação entre as pessoas com dificuldade para obter ajuda sobre o Gatsby.

Reviewer: Aqui a intenção seria descrever informação de acessibilidades, não que elas acessíveis, não?
```suggestion
Essa página foi criada para coletar informações de acessibilidade sobre todo o ecossistema Gatsby em um único lugar, melhorando os canais de comunicação entre as pessoas com deficiências para obterem ajuda sobre o Gatsby.
```

## Produtos e serviços Gatsby

> Gatsbyjs.org é o site oficial do framework de codigo aberto Gatsby; que inclui documentação, tutoriais e guias de como e porque desenvolver com Gatsby, bem como um blog, uma galeria de projetos, e informações sobre como contribuir com o projeto.   

Reviewer 2: ```suggestion
Gatsbyjs.org é o site oficial do framework de código aberto Gatsby; que inclui documentação, tutoriais e guias de como e porque desenvolver com Gatsby, bem como um blog, uma galeria de projetos, e informações sobre como contribuir com o projeto.   
```

Gatsbyjs.com é a página de negócios da Gatsby, Inc. a startup que desenvolve o Gatsby, incluindo nosso primeiro produto, Gatsby Preview: um serviço em nuvem que permite que membros de um time colaborem no desenvolvimento de um site Gatsby.

## Fornecendo feedback e recebendo ajuda sobre acessibilidade

> Nós nos esforçamos para fazer esses sites e o framework Gatsby por si mesmo o mais acessivel possivel. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compativeis. Nos vamos ser os primeiros a adimitir que o ecossistema Gatsby é um trabalho em andamento, estando sembre abertos a todo tipo de feedback para fazer as coisas da melhor forma possivel.

Reviewer 2: ```suggestion
Nós nos esforçamos para fazer esses sites e o framework Gatsby por sí mesmo o mais acessível possível. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compatíveis. Nos vamos ser os primeiros a adimitir que o ecossistema Gatsby é um trabalho em andamento, estando sembre abertos a todo tipo de feedback para fazer as coisas da melhor forma possível.
```

> Nós nos esforçamos para fazer esses sites e o framework Gatsby por sí mesmo o mais acessível possível. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compatíveis. Nos vamos ser os primeiros a adimitir que o ecossistema Gatsby é um trabalho em andamento, estando sembre abertos a todo tipo de feedback para fazer as coisas da melhor forma possível.

Reviewer 2: ```suggestion
Nós nos esforçamos para fazer esses sites e o framework Gatsby por sí mesmo o mais acessível possível. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compatíveis. Nós vamos ser os primeiros a admitir que o ecossistema Gatsby é um trabalho em andamento, estando sembre aberto a todo tipo de feedback para fazer as coisas da melhor forma possível.
```

Para contatar nosso time principal com suas dicas sobre acessibilidade ou desafios, por favor [envie uma issue no GitHub](https://github.com/gatsbyjs/gatsby/issues/new/choose).

> Como outra alternativa, convidamos você para entrar em contato diretamente com Marcy Sutton, Chefe da divisao de aprendizado no Gatsby: [marcy@gatsbyjs.com](mailto:marcy@gatsbyjs.com)

Reviewer 2: ```suggestion
Como outra alternativa, convidamos você para entrar em contato diretamente com Marcy Sutton, Chefe da divisão de aprendizado no Gatsby: [marcy@gatsbyjs.com](mailto:marcy@gatsbyjs.com)
```

## Compilando com Gastby

> Para aprender como compilar um site acessivel com Gatsby, visite nosso guia [Fazendo seu site acessivel](/docs/making-your-site-accessible/). Contribuições sao muito bem-vindas a medida que a página evolui.

Reviewer 2: ```suggestion
Para aprender como compilar um site acessível com Gatsby, visite nosso guia [Fazendo seu site acessível](/docs/making-your-site-accessible/). Contribuições sao muito bem-vindas a medida que a página evolui.
```

> Para aprender como compilar um site acessível com Gatsby, visite nosso guia [Fazendo seu site acessível](/docs/making-your-site-accessible/). Contribuições sao muito bem-vindas a medida que a página evolui.

Reviewer 2: ```suggestion
Para aprender como compilar um site acessível com Gatsby, visite nosso guia [Fazendo seu site acessível](/docs/making-your-site-accessible/). Contribuições são muito bem-vindas a medida que a página evolui.
```

## Plataformas, produtos e serviços tercerizados.

> O fluxo de trabalho padrão da Gatsby envolve produtos e serviços tercerizados, tais como fornecimento de Sistemas de Gerenciamento de Conteudo, e gerenciamento e desenvolvimento de sites com plataformas de hospedagem. Algumas dessas plataformas parecem ter sido projetadas sem ter a acessibilidade em mente. Nós encorajamos esses fornecedores/desenvolvedores a melhorarem a acessibilidade de forma conjunta conosco.

Reviewer 2: ```suggestion
O fluxo de trabalho padrão da Gatsby envolve produtos e serviços terceirizados, tais como fornecimento de Sistemas de Gerenciamento de Conteúdo, e gerenciamento e desenvolvimento de sites com plataformas de hospedagem. Algumas dessas plataformas parecem ter sido projetadas sem ter a acessibilidade em mente. Nós encorajamos esses fornecedores/desenvolvedores a melhorarem a acessibilidade de forma conjunta conosco.
```

> Nossas atuais recomendações para acessibilidade em plataformas CMS são;

Reviewer: ```suggestion
Nossas atuais recomendações para acessibilidade em plataformas CMS são:
```

- WordPress com o plugin Classic Editor.
- Drupal

Para gerenciar e desenvolver sites Gatsby com ferramentas de acessibilidade, nós recomendamos:

- Netlify

> Para fornecer feedback para serviços de terceiros, voce pode contatar os fornecedores diretamente ou [escreva para nós](mailto:marcy@gatsbyjs.com), e nós vamos fazer nosso melhor para passar a informação adiante. Nós também nos interessamos em escutar sobre seu sucesso com plataformas de terceiros!

Reviewer 2: ```suggestion
Para fornecer feedback para serviços de terceiros, você pode contatar os fornecedores diretamente ou [escreva para nós](mailto:marcy@gatsbyjs.com), e nós vamos fazer nosso melhor para passar a informação adiante. Nós também nos interessamos em escutar sobre seu sucesso com plataformas de terceiros!
```

## Pesquisas e projetos de busca

> O time Gatsby conduz pesquisar e projetos de busca de tempos em tempos. Nós damos alto valor para o feedback vindo de todos os usuarios Gatsby, e principalmente aos com experiência aplicavel nas pesquisas que estão sendo conduzidas.

Reviewer 2: ```suggestion
O time Gatsby conduz pesquisas e projetos de busca de tempos em tempos. Nós damos alto valor para o feedback vindo de todos os usuários Gatsby, e principalmente aos com experiência aplicável nas pesquisas que estão sendo conduzidas.
```

---

> titulo: Declaração de acessibilidade

Reviewer 3: ```suggestion
title: Declaração de acessibilidade
```

---

> _Ultima atualização: Abril 8, 2019_

Reviewer 3: ```suggestion
_Última atualização: Abril 8, 2019_
```

> Nos queremos que todo usuario de Gatsby sinta-se bemvindo e sinta a experiencia de recompensa: isso inclui desenvolvedores e usuarios com dificuldades em sites, documentações e recursos do Gatsby.

Reviewer 3: ```suggestion
Nos queremos que todo usuário do Gatsby sinta-se bem-vindo e sinta a experiência de recompensa: isso inclui desenvolvedores e usuários com dificuldades especiais em sites, documentações e recursos do Gatsby.
```

> Essa pagina foi criada para coletar de forma simples informação sobre todo o ecosistema Gatsby em um unico lugar, melhorando os canais de comunicação entre as pessoas com dificuldade para obter ajuda sobre o Gatsby.

Reviewer 3: ```suggestion
Essa página foi criada para coletar de forma simples informações sobre todo o ecossistema Gatsby em um único lugar, melhorando os canais de comunicação entre as pessoas com dificuldade para obter ajuda sobre o Gatsby.
```

> ## Produtos e serviços Getsby

Reviewer 3: ```suggestion
## Produtos e serviços Gatsby
```

> Gatsbyjs.org é o site oficial do framework de codigo aberto Gatsby; que inclui documentação, tutoriais e guias de como e porque desenvolver com Gatsby, bem como um blog, uma galeria de projetos, e inforções sobre como contribuir com o projeto.   

Reviewer 3: ```suggestion
Gatsbyjs.org é o site oficial do framework de codigo aberto Gatsby; que inclui documentação, tutoriais e guias de como e porque desenvolver com Gatsby, bem como um blog, uma galeria de projetos, e informações sobre como contribuir com o projeto.   
```

> Gatsbyjs.com é a pagina de negocios da Gatsby, Inc. a startup que desenvolve o Gatsby, incluindo nosso primeiro produto, Gatsby Preview: um serviço em nuvem que permite que membros de um time colaborem no desenvolvimento de um site Gatsby.

Reviewer 3: ```suggestion
Gatsbyjs.com é a página de negócios da Gatsby, Inc. a startup que desenvolve o Gatsby, incluindo nosso primeiro produto, Gatsby Preview: um serviço em nuvem que permite que membros de um time colaborem no desenvolvimento de um site Gatsby.
```

> ## Fornecendo feedback e recebendo ajuda sobre a acessibilidade

Reviewer 3: ```suggestion
## Fornecendo feedback e recebendo ajuda sobre acessibilidade
```

> Nós nos esforçamos para fazer esses sites e o framework Gatsby por si mesmo o mais acessivel possivel. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compativeis. Nos vamos ser os primeiros a adimitir que o ecosistema Gatsby é um trabalho em andamento, estando sembre abertos a todo tipo de feedback para fazer as coisas da melhor forma possivel.

Reviewer 3: ```suggestion
Nós nos esforçamos para fazer esses sites e o framework Gatsby por si mesmo o mais acessivel possivel. Nosso objetivo é atender todas as recomendações do [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), com o qual somos ainda parcialmente compativeis. Nos vamos ser os primeiros a adimitir que o ecossistema Gatsby é um trabalho em andamento, estando sembre abertos a todo tipo de feedback para fazer as coisas da melhor forma possivel.
```

Para contatar nosso time principal com suas dicas sobre acessibilidade ou desafios, por favor [envie uma issue no GitHub](https://github.com/gatsbyjs/gatsby/issues/new/choose).

> Como outra alternativa, convidamos voce para entrar em contato diretamente com Marcy Sutton, Chefe da divisao de aprendizado no Gatsby: [marcy@gatsbyjs.com](mailto:marcy@gatsbyjs.com)

Reviewer 3: ```suggestion
Como outra alternativa, convidamos você para entrar em contato diretamente com Marcy Sutton, Chefe da divisao de aprendizado no Gatsby: [marcy@gatsbyjs.com](mailto:marcy@gatsbyjs.com)
```

## Compilando com Gastby

> Para aprender como compilar um site acessivel com Gatsby, visite nosso guia [Fazendo seu site acessivel](/docs/making-your-site-accessible/). Contribuições sao muito bem vindas, dessa forma a pagina evolui.

Reviewer 3: ```suggestion
Para aprender como compilar um site acessivel com Gatsby, visite nosso guia [Fazendo seu site acessivel](/docs/making-your-site-accessible/). Contribuições sao muito bem-vindas a medida que a página evolui.
```

## Plataformas, produtos e serviços tercerizados.

> O fluxo de trabalho padrão da Gatsby envolve produtos e serviços tercerizados, tais como fornecimento de Sistemas de Gerenciamento de Conteudo, e genrenciamento e desenvolvimento de sistes com plataformas de hospedagem. Algumas dessas plataformas parecem terem sido projetadas sem ter a acessibilidade em mente. Nós encorajamos esses fornecedores/desenvolvedores a melhorarem a acessibilidade de forma conjunta com nós.

Reviewer 3: ```suggestion
O fluxo de trabalho padrão da Gatsby envolve produtos e serviços tercerizados, tais como fornecimento de Sistemas de Gerenciamento de Conteudo, e gerenciamento e desenvolvimento de sites com plataformas de hospedagem. Algumas dessas plataformas parecem ter sido projetadas sem ter a acessibilidade em mente. Nós encorajamos esses fornecedores/desenvolvedores a melhorarem a acessibilidade de forma conjunta conosco.
```

Nossas atuais recomendações para acessibilidade em plataformas CMS são;

- WordPress com o plugin Classic Editor.
- Drupal

Para gerenciar e desenvolver sites Gatsby com ferramentas de acessibilidade, nós recomendamos:

- Netlify

> Para fornecer feedback para serviços de terceiros, voce pode contatar os fornecedores diretamente ou [escreva para nós](mailto:marcy@gatsbyjs.com), e nós vamos fazer nosso melhor para passar a informação adiante. Nós tambem nos interessamos em escutar sobre seu sucesso com plataformas de terceiros!

Reviewer 3: ```suggestion
Para fornecer feedback para serviços de terceiros, voce pode contatar os fornecedores diretamente ou [escreva para nós](mailto:marcy@gatsbyjs.com), e nós vamos fazer nosso melhor para passar a informação adiante. Nós também nos interessamos em escutar sobre seu sucesso com plataformas de terceiros!
```

## Pesquisas e projetos de busca

> O time Gatsby conduz pesquisar e projetos de busca de tempos em tempos. Nós damos alto valor para o feedback vindo de todos os usuarios Gatsby, e principalmente aos com experiencia aplicavel nas pesquisas que estão sendo conduzidas.

Reviewer 3: ```suggestion
O time Gatsby conduz pesquisar e projetos de busca de tempos em tempos. Nós damos alto valor para o feedback vindo de todos os usuarios Gatsby, e principalmente aos com experiência aplicavel nas pesquisas que estão sendo conduzidas.
```
